go.mod

```diff
@@ -0,0 +1,79 @@
+module github.com/yext/terraform-provider-teamcity
+
+go 1.19
+
+require (
+	github.com/hashicorp/terraform v0.12.2
+	github.com/robfig/cron v1.2.0
+	github.com/yext/go-teamcity v0.5.3-0.20230323164551-08f28b0ad89b
+)
+
+require (
```

Comment:
Should both of these require statements be merged? Or was there a particular reason they were made separate?

Reply:
It looks like that is the expected format for go.mod. The first section lists all direct dependencies used in the code, and the second section has all indirect dependencies. This is also the case for the go.mod file in alpha

Reply:
Looks good to me then, thanks for the explanation!